input/pagecontent/volume-1.md

```diff
@@ -186,19 +186,21 @@ The requests shall use the information in DiagnosticReport.result.derivedFrom.en
 
 A Report Reader that also supports the Series/Study Navigation Option shall be able to construct WADO-RS URLs that confirm to the RetrieveSeries and RetrieveStudy semantics.
 
-### XX.2.4 External Image Display Retrieve Option
+### XX.2.4 External IID Image Display Retrieve Option
 
-The External Image Display Retrieve Option enables actors to display images via an external Image Display actor. See [Use Case #3] (#xx423-use-case-3-consume-and-interact-with-multimedia-report-by-report-reader-with-integrated-invoker-image-display).
+The External IID Image Display Retrieve Option utilizes the Invoke Image Display (IID) Profile to enable the Report Reader to display images via an external Image Display actor. See [Use Case #3] (#xx423-use-case-3-consume-and-interact-with-multimedia-report-by-report-reader-with-integrated-invoker-image-display).
 
-A Report Reader that supports this option shall use Invoke Image Display [RAD-106] to invoke an Image Display which in turn retrieves images from an Image Manager / Image Archive. Report Readers shall translate the embedded image references and endpoints in the DiagnosticReport resource into corresponding Invoke Image Display URLs.
+A Report Reader that supports this option is grouped with an IID Image Display Invoker and shall use the Invoke Image Display [RAD-106] transaction to invoke an external Image Display.  The Report Reader shall translate the embedded image references and endpoints in the DiagnosticReport resource into corresponding Invoke Image Display [RAD-106] URLs.
 
-> Note: For this option to work, the Image Display also needs to support the [IID Invoke Option](#xx27-iid-invoke-option).
+An Image Display that supports this option shall support the Invoke Image Display [RAD-106] transaction as an Image Display.  <<TO DO:  IS THIS TRUE?>>  When the Image Display receives a [RAD-106] request, it retrieves the requested images from an Image Manager / Image Archive.
```

Comment:
I looked at the display requirements in RAD-106. I think it all applies to IMR Image Display. As discussed, I will add a new IHE_IMR viewer type in RAD-106 as well.

Reply:
All other changes look good.